cpp/include/cudf/stream_compaction.hpp

```diff
@@ -319,11 +314,11 @@ cudf::size_type distinct_count(table_view const& input,
  * @param[in] null_handling flag to include or ignore `null` while counting
  * @param[in] nan_handling flag to consider `NaN==null` or not
  *
- * @return number of unique elements
+ * @return number of distinct rows in the table
  */
-cudf::size_type unordered_distinct_count(column_view const& input,
-                                         null_policy null_handling,
-                                         nan_policy nan_handling);
+cudf::size_type distinct_count(column_view const& input,
+                               null_policy null_handling,
+                               nan_policy nan_handling);
 
 /**
  * @brief Count the unique rows in a table.
```

Comment:
Avoid "unique" in `distinct_count` definition:
```suggestion
 * @brief Count the distinct rows in a table.
```
Also fix:
```diff
- * @param[in] input Table whose unique rows will be counted
+ * @param[in] input Table whose distinct rows will be counted
```